From randy.dunlap@oracle.com  Tue Jan  8 14:39:41 2008
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 31 Dec 2007 10:05:34 -0800
Subject: sysfs: make SYSFS_DEPRECATED depend on SYSFS
Cc: akpm <akpm@linux-foundation.org>, gregkh@suse.de
Message-ID: <20071231100534.dcbed808.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Make SYSFS_DEPRECATED depend on SYSFS since files that check
CONFIG_SYSFS_DEPRECATED don't check for CONFIG_SYSFS first.
Also don't prompt user about SYSFS_DEPRECATED if SYSFS=n.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 init/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/init/Kconfig
+++ b/init/Kconfig
@@ -363,6 +363,7 @@ config CGROUP_CPUACCT
 
 config SYSFS_DEPRECATED
 	bool "Create deprecated sysfs files"
+	depends on SYSFS
 	default y
 	help
 	  This option creates deprecated symlinks such as the
